executor/executor_test.go

```diff
@@ -1611,6 +1611,10 @@ func (s *testSuite) TestMathBuiltin(c *C) {
 	result.Check(testkit.Rows("-10 -10"))
 	result = tk.MustQuery("select ceil(t.c_decimal), ceiling(t.c_decimal) from (select cast('-10.01' as decimal(10,1)) as c_decimal) as t")
 	result.Check(testkit.Rows("-10 -10"))
+	result = tk.MustQuery("select floor(18446744073709551615), ceil(18446744073709551615)")
```

Comment:
Should we add some annotation here or split `floor` from this test block since L1595 mentations ` for ceil/ceiling`?